card/mod/navbar/set/self/account_links.rb

```diff
@@ -8,15 +8,20 @@ def ok_to_read
   view :core, cache: :never do
     status_class = Auth.signed_in? ? "logged-in" : "logged-out"
     wrap_with :span, id: "logging", class: status_class do
-      item_links.join " "
+      navbar_items.join "\n"
```

Comment:
starting to think we want an "item_joint" view option.  Maybe "separator" (to use the language of csvs?)